src/Common/src/Interop/Unix/libssl/Interop.OpenSsl.cs

```diff
@@ -250,6 +262,83 @@ private static IntPtr GetSslMethod(bool isServer, long options)
             return method;
         }
 
+        private static int verify_callback(int preverify_ok, IntPtr x509_ctx_ptr)
```

Comment:
Rather than `verify_callback` (which is also the name of the delegate), how about `VerifyClientCertificate`?